src/client.rs

```diff
@@ -138,6 +143,33 @@ fn deserialize_tick(
     }
 }
 
+fn deserialize_entity_mappings(
+    cursor: &mut Cursor<Bytes>,
+    world: &mut World,
+    entity_map: &mut NetworkEntityMap,
+) -> Result<(), bincode::Error> {
+    let array_len: u16 = bincode::deserialize_from(&mut *cursor)?;
+    for _ in 0..array_len {
+        let server_entity = deserialize_entity(cursor)?;
+        let client_entity = deserialize_entity(cursor)?;
+
+        if let Some(entry) = entity_map.to_client().get(&server_entity) {
+            error!("received mapping from {server_entity:?} to {client_entity:?}, but already mapped to {entry:?}");
+            continue;
+        }
```

Comment:
It actually a client code, maybe we should `panic!`?
This should never happen. If it does - the server is programmed wrong.
We should never panic on server, but panicking on client is fine. It could happen even if you misuse bevy.